dts/bindings/pinctrl/nxp,mcux-rt-pinctrl.yaml

```diff
@@ -163,6 +157,18 @@ child-binding:
           Select slew rate for pin. Corresponds to SRE field in IOMUXC peripheral
           0 SRE_0_Slow_Slew_Rate — Slow Slew Rate
           1 SRE_1_Fast_Slew_Rate — Fast Slew Rate
+      nxp,pull:
+        required: true
+        type: string
+        enum:
+          - "no-pull"
+          - "pull-enable"
+        description: |
+          Select pull enable bit. If this bit is selected, than the pin
+          pull/keeper settings take effect. If it is cleared, no pull is applied
+          to the pin.
+          0 PKE_Disable- No pull applied to pin
+          1 PKE_Enable- pull or keeper applied to pin depending on bias,bus-hold
```

Comment:
you could infer this from the existence of `bias-pull-up`, `bias-pull-down` or `bias-bus-hold`, right?

```
pke = DT_PROP(node_id, bias_pull_up) | DT_PROP(node_id, bias_pull_down) | DT_PROP(node_id, bias_bus_hold);
```

Reply:
I could, and originally took this approach, but the concern with inferring the value of `pke` was the same issue I outlined in [this comment below](https://github.com/zephyrproject-rtos/zephyr/pull/42238#issuecomment-1031255945), namely that the "default" value for `PKE` is almost always 1, so the user might end up with unexpected affects if they level all boolean properties unselected on a pin. My plan right now is to drop `bias-bus-hold` and only set PKE to 0 if `bias-disable` is selected.